examples/lighting-app/esp32/README.md

```diff
@@ -154,10 +154,17 @@ After commissioning is successful, press Enter in requestor device console and
 type below query.
 
 ```
->matter ota query 1 12345 0
+>matter ota query 1 12346 0
+```
+
+Alternate way to query an OTA image using chip-tool
+
+```
+./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12346 0 0 0 12345 0
```

Comment:
Please see if you want to remove this to avoid confusion, as per our discussion offline

Reply:
Removed this option.